org.freeorion.FreeOrion.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.freeorion.FreeOrion
+command: freeorion
+
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=${FLATPAK_DEST} --with-python=/usr/bin/python3
+           --with-libraries=filesystem,iostreams,locale,log,serialization,system,thread,python
+      - ./b2 --reconfigure -j ${FLATPAK_BUILDER_N_JOBS} toolset=gcc threading=multi
+          variant=release debug-symbols=off link=shared,static runtime-link=shared
```

Comment:
That seems more complicated than existing boost builds. Is there any reason to include both shared and static libraries?

Reply:
Removed static libs.

Reply:
The other libraries are explicitly required for the game.